test-case-reporting/app.ts

```diff
@@ -76,6 +76,13 @@ function enumerateTestRun(
   return {testRun, index};
 }
 
+function lowerCaseStatus({status, ...rest}: TestRun): any {
```

Comment:
Are you going to be using Handlebars or Mustache?

Reply:
I'm keeping mustache for now.